README.md

```diff
@@ -0,0 +1,20 @@
+﻿## Testing
+
+### Prerequisites
+
+-   node 12.20.0
+-   yarn 1.22.0
+
+### Installation
+
+-   `yarn install`
+
+### Verification
+
+-   Verifying all the contracts:
+    -   `yarn test` (quick testing)
+    -   `yarn coverage` (full coverage)

```

Comment:
there's no coverage here, no?

Reply:
`solidity coverage` is an integral part of HardHat as far as I know, so this should work I suppose.

Reply:
ok